numpy/core/einsumfunc.py

```diff
@@ -169,7 +169,7 @@ def _optimal_path(input_sets, output_set, idx_dict, memory_limit):
     Examples
     --------
     >>> isets = [set('abd'), set('ac'), set('bdc')]
-    >>> oset = set('')
+    >>> oset = {''}
```

Comment:
This change is not correct - `set('')` is the empty set, not a set containing the empty string